### PR TITLE
Dala 5047 Update SFDR questionnaire

### PR DESCRIPTION
**Pull request by juliacialkowski**
# Pull Request \<Title>
`<Description here>`
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The GitHub Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by GitHub. 
- [x] The PR has been peer-reviewed
  - [x] The code has been manually inspected by someone who did not implement the feature
  - [ ] If this PR includes work on the frontend, at least one `@ts-nocheck` is removed. Additionally, there should not be any `@ts-nocheck` in files modified by this PR. If no `@ts-nocheck` are left: Celebrate :tada: :confetti_ball: type-safety and remove this entry. 
- [x] The PR actually implements what is described in the JIRA-Issue
- [ ] At least one test exists testing the new feature
  - [ ] If you have created new test files, make sure that they are included in a test container and actually run in the CI
- [x] At least 3 consecutive CI runs are successfully executed to ensure that there are no flaky tests.
- [ ] Documentation is updated as required
- [ ] The automated deployment is updated if required
- [ ] If there was a database entity class added, there must also be a migration script for creating the corresponding database if flyway is already used by the service
- [ ] IF there are changes done to the framework data models or to a database entity class, the following steps were completed in order
  - [ ] A fresh clone of dataland.com is generated (see Wiki page on "OTC" for details)
  - [ ] The feature branch is deployed to clone with `Reset non-user related Docker Volumes & Re-populate` turned off
  - [ ] It's verified that the CD run is green
  - [ ] The new feature is manually used/tested/observed on the clone server. Testing of the new feature should (also) be done by a second, independent reviewer from the dev team
  - [ ] The feature branch is deployed to dev1 with `Reset non-user related Docker Volumes & Re-populate` turned on, and it's verified that the CD run is green  
- [x] ELSE, the new version is deployed to the dev server "dev1" using this branch
  - [x] Run with setting `Reset non-user related Docker Volumes & Re-populate` turned on 
  - [x] It's verified that this version actually is the one deployed (check gitinfo for branch name and commit id!)
  - [x] It's verified that the CD run is green
  - [x] The new feature is manually used/tested/observed on dev server. Testing of the new feature should (also) be done by a second, independent reviewer from the dev team
- [x] Confirm that the latest version of the feature is deployed to the dev server "dev1" using this branch (no longer enforced by GitHub)

### CODE REVIEW

File: dataland-framework-toolbox/inputs/sfdr/sfdr.csv [hunk 1]
```diff
@@ -16,97 +16,100 @@ Field Identifier,Category,Sub-Category,Field Name,Tooltip - upload page,Tooltip
...
-109,Social,Anti-corruption and anti-bribery,Cases of Insufficient Action against Bribery and Corruption,Identified insufficiencies in actions taken to address breaches in procedures and standards of anti-corruption and anti-bribery, ,Integer,"Allowed Range: [0, INF]", ,Extended,,,
-110,Social,Anti-corruption and anti-bribery,Reported Convictions Of Bribery and Corruption,Number of reported convictions for violations of anti-corruption and anti-bribery laws, ,Integer,"Allowed Range: [0, INF]", ,Extended,,,
-111,Social,Anti-corruption and anti-bribery,Total Amount Of Reported Fines Of Bribery and Corruption,Amount of fines for violations of anti-corruption and anti-bribery laws, ,Currency,"Allowed Range: [0, INF]", ,Extended,,,
+18,Environmental,Greenhouse gas emissions ,Scope 4 GHG emissions,Scope 4 carbon emissions refer to emissions avoided when a product is used as a substitute for other goods or services that fulfill the same functions but have a lower carbon intensity., ,Number,,Tonnes,Extended,,,
```
Comment (jenniferlechner) [inline dataland-framework-toolbox/inputs/sfdr/sfdr.csv:19]: Anführungszeichen für den Text "Scope 4 carbon emmisions refer to ..."

File: dataland-framework-toolbox/inputs/sfdr/sfdr.csv [hunk 2]
```diff
@@ -16,97 +16,100 @@ Field Identifier,Category,Sub-Category,Field Name,Tooltip - upload page,Tooltip
...
+26,Environmental,Greenhouse gas emissions ,GHG intensity - scope 4,Tonnes of Scope 4 GHG emissions / million of the revenue (in the same currency as the total revenue),,Number,,Tonnes / €M Revenue,Extended,,,
+27,Environmental,Greenhouse gas emissions ,Fossil Fuel Sector Exposure,"Do you derive any revenues from exploration, mining, extraction, production, processing, storage, refining or distribution, including transportation, storage and trade, of fossil fuels as defined in Article 2, point (62), of Regulation (EU) 2018/1999 of the European Parliament and of the Council? See also Regulation, Annex I, top (5).","(Part of) revenues derived from exploration, mining, extraction, production, processing, storage, refining or distribution, including transportation, storage and trade, of fossil fuels as defined in Article 2, point (62), of Regulation (EU) 2018/1999 of the European Parliament and of the Council? See also Regulation, Annex I, top (5).",Yes/No, ,,Extended,,,
+28,Environmental,Greenhouse gas emissions ,Financed scope 1 and scope 2 emissions,The sum of scope 1 and scope 2 emissions of financed companies,,Number,,Tonnes ,Extended,,,
```
Comment (jenniferlechner) [inline dataland-framework-toolbox/inputs/sfdr/sfdr.csv:29]: Anführungszeichen für den Text "The sum of scope..."

File: dataland-framework-toolbox/inputs/sfdr/sfdr.csv [hunk 3]
```diff
@@ -16,97 +16,100 @@ Field Identifier,Category,Sub-Category,Field Name,Tooltip - upload page,Tooltip
...
+29,Environmental,Greenhouse gas emissions ,Financed scope 3 emissions,The scope 3 emissions of financed companies,,Number,,Tonnes ,Extended,,,
```
Comment (jenniferlechner) [inline dataland-framework-toolbox/inputs/sfdr/sfdr.csv:30]: ebenfalls: Anführungszeichen für den Text "The scope 3 ..."

**Review by jenniferlechner — COMMENTED** [review]
Do we need to update documentation? For Taxonomy, we have the excel files on our Wiki? Should we add the new SFDR ones as well?

File: dataland-framework-toolbox/inputs/sfdr/sfdr.csv (hunk 1 above)
Comment (juliacialkowski) [reply dataland-framework-toolbox/inputs/sfdr/sfdr.csv:19]: Die Anfürhungszeichen werden durch die Framework Toolbox automatisch gesezt, wenn in dem Text, wie hier zB in dem Tooltip, ein Komma enthalten ist. Das Komma trennt normalerweise die Excel Einträge in einer Zeile. Wenn wir noch ein Komma in dem Tooltip selbst haben, wird es in "" Zeichen gesetzt.

File: dataland-framework-toolbox/inputs/sfdr/sfdr.csv (hunk 2 above)
Comment (juliacialkowski) [reply dataland-framework-toolbox/inputs/sfdr/sfdr.csv:29]: s.o.

File: dataland-framework-toolbox/inputs/sfdr/sfdr.csv (hunk 3 above)
Comment (juliacialkowski) [reply dataland-framework-toolbox/inputs/sfdr/sfdr.csv:30]: s.o.

**Pull request merged**